test/fail_compilation/fail109.d

```diff
@@ -1,7 +1,7 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/fail109.d(12): Error: enum member fail109.Bool.Unknown initialization with (Bool.True + 1) causes overflow for type 'bool'
+fail_compilation/fail109.d(12): Error: enum member fail109.Bool.Unknown initialization with `Bool.True+1` causes overflow for type `bool`
```

Comment:
If D had error message codes, the test could just check for the correct code instead of an exact output string... it'd make changing these more robust since you could actually rely on the test and not be changing them so frequently.

We really should add them.